canonical-websites/insights.ubuntu.com
# Update run script

## Done
Updated the `./run` script to the latest version

## QA
- Run `./run -version` and you should get `Generated from generator-canonical-webteam@2.1.1`
- Run `./run clean` and it should clean all built files
- Run `./run` and it should reinstall all things
